src/jit/lower.h

```diff
@@ -55,27 +55,63 @@ class Lowering : public Phase
             LowerNode(newNode);
         }
     }
+    void LowerRange(GenTree* firstNode, GenTree* lastNode)
+    {
+        for (GenTree* node = firstNode; node != lastNode->gtNext; node = node->gtNext)
+        {
```

Comment:
@pgavlin - how would you suggest handling the pattern (e.g. in flowgraph.cpp) where:
- A couple of instructions are inserted above instruction X
- The new instructions, *plus* instruction X need to be relowered
I don't want to have to delete the original instruction and then add it to a range, and then insert it into the block to do this, as that is wasted work.